build.gradle

```diff
@@ -71,6 +78,13 @@ sourceSets
         }
         resources.srcDir file('src/integrationTest/resources')
     }
+    main
+    {
+        java
+        {
+            srcDir file('gen/main/java')
```

Comment:
How about calling that folder `src/generated/java`? It would fall nicely with the other `src/main/java`, `src/test/java` and `src/integrationTest/java`...

Reply:
See note regarding ECLIPSE_README above